docs/source/Doxyfile

```diff
@@ -162,7 +162,7 @@ FULL_PATH_NAMES        = YES
 # will be relative from the directory where doxygen is started.
 # This tag requires that the tag FULL_PATH_NAMES is set to YES.
 
-STRIP_FROM_PATH        = ..
+STRIP_FROM_PATH        = ../../BattleNetwork
```

Comment:
This is different than what was being told to exhale, which is why the file paths are so strange.  Should be good now.